case/src/main/kotlin/com/ritense/case/web/rest/CaseInstanceResource.kt

```diff
@@ -41,6 +43,8 @@ class CaseInstanceResource(
         @RequestBody searchRequest: SearchWithConfigRequest,
         pageable: Pageable
     ): ResponseEntity<Page<CaseListRowDto>> {
+        val tenantId = tenantResolver.getTenantId()
+        searchRequest.tenantId = tenantId
```

Comment:
You could also directly assign the tenantId from the resolver to the search request instead of first setting a local variable which is only used once.